index.html

```diff
@@ -254,7 +264,15 @@ <h1>Examples:</h1>
                                                   <td>20181124</td>
                                                   <td><a href="data/demos/LookTest_20181124.aia">LookTest_20181124.aia</a></td>
                                                 </tr>
-
+						<tr>
+                                                  <td>PosenetTest</td>
+                                                  <td>
+                                                    A project demonstrating Posenet using an extension.
```

Comment:
Clarifying question: is this the app I made or does this make use of your newer canvas feature?

Reply:
This is an updated version that makes use of the newer canvas feature.

Reply:
Great!